README.md

```diff
@@ -1 +1,97 @@
 # deis-cleanup
+
+## Usage
+### Environment variables
+Script can be configured by setting environement variables. It can by passed to
+docker run (`docker run -e "ENV_NAME=value" ...`).
```

Comment:
imho it should be "Configuration" header with the following paragraph (suggestion):
 
### Configuration

Use environment variables to pass the configuration. For example:
```bash
docker run --env "FOO=bar"
```

#### Possible options:
...